src/routes/contacts.route.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const ContactsController = require('../controllers/contacts.controller')
+
+module.exports = [{
+  method: ['GET', 'POST'],
+  path: '/search',
```

Comment:
If the route for this page is /search I would find it less confusing if the corresponding controller and route file was renamed to search.controller.js and search.route.js, especially as we already have a 'contact' route. 

I would accept the point though that these pages are only POCs and therefore likely to be deleted anyway :)